README.md

```diff
@@ -10,10 +10,6 @@ Kompendium is a simple, fast and elegant tool for generating documentation for c
 
 Just type everything in Markdown, from your code blocks to description texts of example components. Then enjoy a lightning fast responsive compendium of your components, with component API and live playground.
 
-### Please note
-
-Kompendium is still in development. Consider this an early preview!
```

Comment:
I agree, but this is very much up to @jgroth to decide.

Also, we might want to bump to v1.0.0 then?

Reply:
We have been using Kompendium forever kind of. So if there has been any show-stopper issues, we would have discovered them.

If we want external help, and also if we want people to actually use Kompendium in their project (to then contribute to it), then including this note is just super bad, strategically. 

😅 How would you yourself do, if you came to an open-source project which basically says right in the beginning: 
> well... you better perhaps not use this package, if you don't want any headaches

Also, Kompendium is amazing as is!

Reply:
Again, I completely agree! But it's still up to Johan to make the decision 😊

Reply:
yeah I wrote that for Johan :P